client/config/webpack.common.js

```diff
@@ -47,7 +47,11 @@ module.exports = {
                 }
             }, {
                 test: /\.css$/,
-                use: ["style-loader", "css-loader"]
+                use: [
+                    'style-loader',
+                    { loader: 'css-loader', options: { importLoaders: 1 } },
+                    'postcss-loader'
+                    ]
```

Comment:
Reduce indentation of the closing bracket.

Reply:
done